provider/linear/index.ts

```diff
@@ -49,14 +72,27 @@ const linearIssues: Provider<Settings> = {
             return []
         }
 
-        const linearClient = getLinearClient(settingsInput)
-
-        const issue = await linearClient.issue(issueId)
-        const comments = await issue.comments()
+        const query = `
+            query Issue($id: String!) {
+                issue(id: $id) {
+                    title
+                    description
+                    comments {
+                        nodes {
+                            body
+                        }
+                    }
+                }
+            }
+        `
+        const variables = { id: issueId }
+        const data = await linearApiRequest(query, variables, settingsInput)
+        const issue = data.data.issue as Issue
+        const comments = issue.comments?.nodes as Comment[]
 
         const content = xmlBuilder.build({
             description: issue.description || '',
-            comments: comments.nodes.map(comment => comment.body).join('\n'),
+            comments: comments.map(comment => comment.body).join('\n'),
```

Comment:
Lets do:
```
xmlBuilder.build({
  linear_issue: {
    description: issue.description || '',
    comments: { 
      comment: comments.map(comment => comment.body).join('\n')
    }
  }
})
```

This will generate:
```
<linear_issue>
  <description>...</description>
  <comments>
    <comment>...</comment>
    <comment>...</comment>
  </comments>
<linear_issue>
```